src/main/play-doc/operation/CLI.md

```diff
@@ -79,14 +79,16 @@ The CLI is able to integrate with the DC/OS CLI e.g. `dcos conduct info` will re
 
 ## HTTP Basic Authentication
 
-To enable HTTP Basic Authentication, provide the following settings file in the `~/.conductr/settings.conf`.
+Note: From a cluster setup perspective, HTTP Basic Authentication for the ConductR CLI can be setup and enforced as described in the section in the section [Securing the ConductR CLI with Basic Authentication](DynamicProxyConfiguration#Securing-the-ConductR-CLI-with-Basic-Authentication).
```

Comment:
Also, grammar correction:

- `in the section` instead of `in the section in the section`
- `ConductR CLI can be set up` instead of `ConductR CLI can be setup`